heclib/heclib_c/src/Dss6/lockdss.c

```diff
@@ -66,16 +66,16 @@ void lockdss_(int *ihandle, int *mode, int *position, int *nbytes, int *istat)
 
 void lockdss_(int *ihandle, int *mode, int *position, int *nbytes, int *istat)
 {
-	off64_t lbytes;
-	off64_t lposition;
+	off_t lbytes;
```

Comment:
It's rarely a good idea go from more to less specific. Why did the off64_t not work? If it was MACOS specific it should be wrapped in #if defined __APPLE__

Reply:
Gotcha, I'll update it 😁 👍

Reply:
PR updated with macro to use [off_t, lseek, lockf] instead of the '64' version for macOS